### DISCUSSION TITLE
How to disable debug in the console?

### DISCUSSION THREAD

**Discussion by 07skat** [MontFerret/ferret#669] [Q&A] [answered]
How can I disable debugging in the console in this [embedded example](https://github.com/MontFerret/ferret/blob/master/examples/embedded.go)?

When I run the example, the first line I get is something like this:
`{"level":"debug","id":"8bb8adc2-dbba-49f4-ac79-b18180bf993f","time":"2019-04-18T13:17:50+05:00","user-agent":"","message":"using User-Agent"}`

How to disable this output?

**Comment by ziflex** [comment, marked as answer]
Hey, you can pass your custom io.Writer like this:

```go
out, err := program.Run(ctx, runtime.WithLog(os.Stdout))
```

**Comment by 07skat** [comment]
Thanks to this, I found a solution.
